app/src/main/java/com/fstyle/structure_android/screen/userlist/UserListContract.java

```diff
@@ -0,0 +1,34 @@
+package com.fstyle.structure_android.screen.userlist;
+
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.source.remote.config.error.BaseException;
```

Comment:
ERROR: Unused import: com.fstyle.structure_android.data.source.remote.config.error.BaseException. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports